Doc/library/smtplib.rst

```diff
@@ -556,34 +556,33 @@ This example prompts the user for addresses needed in the message envelope ('To'
 and 'From' addresses), and the message to be delivered.  Note that the headers
 to be included with the message must be included in the message as entered; this
 example doesn't do any processing of the :rfc:`822` headers.  In particular, the
-'To' and 'From' addresses must be included in the message headers explicitly. ::
+'To' and 'From' addresses must be included in the message headers explicitly::
 
    import smtplib
 
    def prompt(prompt):
        return input(prompt).strip()
```

Comment:
This `prompt` parameter shadows a name from the outer scope, the `prompt` function.

If we're tidying this example up, shall we "fix" this too?

Reply:
I think so. I'll change the variable name of the prompt, or I can just remove the function altogether and use an inlined version.